### DISCUSSION TITLE
Kade Engine won't compile

### DISCUSSION THREAD

**Discussion by prockscac** [KadeArchive/Kade-Engine#348] [Q&A] [answered]
When ever I compile the latest version of Kade Engine It says 'clang: error: linker command failed with exit code 1 (use -v to see invocation)
Error: Error while running command
xcrun --sdk macosx clang++ -o ApplicationMain -framework Cocoa -m64 -framework Security 
"lol my name and stuff goes here' Kade-Engine-1.4.2 %  and yes I used lime test mac/windows/linux -v and it still said the same thing.

**Comment by puyoxyz** [comment]
What platform are you building on? Did you try building to html5 (and does that work, if you did?)

**Reply by prockscac** [reply]
I use terminal if that's what your talking about

**Reply by Kade-github** [reply]
No she meant what OS you are using and what you are targeting.

**Reply by prockscac** [reply]
i'm using MacBook 64 bits.

**Comment by puyoxyz** [comment, marked as answer]
We don't support macOS right now. I'll see if it's actually broken (or if you made a mistake) and if it is then I'll make an issue.

**Reply by prockscac** [reply]
Oh okay thanks!

**Reply by puyoxyz** [reply]
It seems the problem was that you don't have Xcode installed -- I'll update the documentation to reflect this and set up CI builds targeting macOS.

**Reply by isakube** [reply]
i literally made an entire video proving you wrong and then you figured out the problem lol https://www.youtube.com/watch?v=9DG5nMfHCSk

does this mean my commit can get un-reverted now just btw?
oh it did

**Reply by puyoxyz** [reply]
> i literally made an entire video proving you wrong and then you figured out the problem lol https://www.youtube.com/watch?v=9DG5nMfHCSk
> 
> does this mean my commit can get un-reverted now just btw?
> oh it did

I meant that it wasn't tested so we didn't *officially* support it. (now we do)
